packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -134,10 +141,10 @@ class SlidePanel extends React.Component {
         data-slide-panel-panel-position={panelPosition}
         data-slide-panel-panel-size={panelSize}
       >
-        <div className={cx('main')}>
+        <div className={cx('main')} tabIndex="-1" ref={this.setMainNode}>
```

Comment:
We can set the MainNode value here itself inline. instead of having a separate event handler.

Reply:
Updated [reference for main node](https://github.com/cerner/terra-framework/pull/584/commits/eebe761af198b2489aed3529bea7870d57599092#diff-810d8b37f7659eab5a75af63e03e20e8R139)

Reply:
You could also switch over to the createRef API that was introduced a few versions back: https://reactjs.org/docs/refs-and-the-dom.html#creating-refs

I'm finding myself using that pretty much exclusively at this point.

Reply:
updated to [CreateRef](https://github.com/cerner/terra-framework/pull/584/commits/507edc3acdddf9a8b03db274d1f1e9028f95387d)